Tests/SentryTests/Integrations/Performance/IO/SentryNSDataTrackerTests.swift

```diff
@@ -105,6 +108,46 @@ class SentryNSDataTrackerTests: XCTestCase {
         assertSpanDuration(span: span, expectedDuration: 4)
         assertDataSpan(span, path: fixture.filePath, operation: SENTRY_FILE_WRITE_OPERATION, size: fixture.data.count)
     }
+
+    func testWriteAtomically_CheckTransaction_DebugImages() {
```

Comment:
`m`: Shouldn't this teat also call `assertDataSpan`?

Reply:
Not really necessary